--- qpid-proton-0.35.0/CMakeLists.txt~	2021-06-25 15:42:55.000000000 +0200
+++ qpid-proton-0.35.0/CMakeLists.txt	2021-10-13 21:57:43.540058477 +0200
@@ -176,10 +176,6 @@
 set (LINK_LTO_Clang "-flto=thin")
 set (LINK_LTO_MSVC "/LTCG")
 
-set (WERROR_GNU "-Werror")
-set (WERROR_Clang "-Werror")
-set (WERROR_MSVC "")
-
 set (COMMON_WARNING_GNU "-Wall -pedantic-errors")
 set (COMMON_WARNING_Clang "-Wall -pedantic")
 set (COMMON_WARNING_MSVC "")
