--- qpid-proton-0.32.0/c/tests/threaderciser.c.orig	2020-10-13 19:03:31.807418794 +0200
+++ qpid-proton-0.32.0/c/tests/threaderciser.c	2020-10-13 19:03:34.654070039 +0200
@@ -94,7 +94,7 @@
   char msg[256];
   char *i = msg;
   char *end = i + sizeof(msg);
-  i += assert_no_err(snprintf(i, end-i, "(%lx) ", (uintptr_t) pthread_self()));
+  i += assert_no_err(snprintf(i, end-i, "(%tx) ", (uintptr_t) pthread_self()));
   if (i < end) {
     va_list ap;
     va_start(ap, fmt);
